### ISSUE TITLE
Release notes for 7.2.0

### ISSUE THREAD

**Issue by mjgiarlo** [projecthydra/sufia#2491]
## Authority change
```
cp config/authorities/rights.yml config/authorities/licenses.yml
```

## Generator to run:
```
$ rails generate jquery:datatables:install bootstrap3
```

## Unreleased gem
```
gem 'flipflop', git: 'https://github.com/jcoyne/flipflop.git', branch: 'hydra'
```

https://github.com/voormedia/flipflop/issues/4
https://github.com/voormedia/flipflop/issues/8
https://github.com/voormedia/flipflop/issues/10

# New flipflop migration
```
$ bin/rails sufia:install:migrations
$ rake db:migrate
```

# Changed default layout

If you are using the sufia-one-column layout it has been moved: `app/views/layouts/sufia-one-column.html.erb → app/views/layouts/curation_concerns/1_column.html.erb`

Check your application.rb for `layout 'sufia-one-column'` and replace it with `with_themed_layout '1_column'`.

In addition if you have overridden sufia-one-column in your application you should move your override to the new location.

# Remove unneeded table (optional)
Sufia no longer includes `acts_as_follower` so if you're upgrading, and you don't want to keep this data, you can add a migration to remove the table:

```
class DropFollows < ActiveRecord::Migration
  def self.up
    drop_table :follows
  end
end
```

## Ordering changes
#2479 brought Rails 5 support to Sufia, and also brought support for RDF 2 which ends a long-standing bug in ActiveFedora that returns RDF statements in a predictable, and not directly alterable, order. In other words, if an item has more than one title, the first title may change whenever the item is loaded.

A workaround for preserving the old behavior is to add `gem 'rdf', '~> 1.99'` to your `Gemfile`, and ordering will be preserved.

**Comment by jcoyne** [comment]
Blocked by 
- [ ] Release of AF 10.3 (which is blocked by https://github.com/projecthydra/active_fedora/pull/1140)
- [ ] https://github.com/projecthydra/curation_concerns/pull/912

**Comment by mjgiarlo** [comment]
@jcoyne Now we just need a release of AF 10.3 -- those two PRs have been merged.

**Comment by jcoyne** [comment]
AF 10.3.0.rc1 has been released and it seems to work, so I'll push 10.3.0.

**Comment by mjgiarlo** [comment]
Need to include the new db migration that drops the `follows` table in the upgrade process. New generator?

**Comment by jcoyne** [comment]
I don't think we need to drop that table. Someone might want to keep it locally.  However, we should remove the up migration if it's no longer needed.

**Comment by jcoyne** [comment]
See https://github.com/projecthydra/sufia/pull/2677

**Comment by cam156** [comment]
Created a draft of the release notes based on the description: https://github.com/projecthydra/sufia/releases/edit/untagged-5d369a647834007fc876

**Comment by weiweishi** [comment]
New flipflop migration

```bin/rails sufia:install:migrations``` should be ```rake sufia:install:migrations```

**Comment by jcoyne** [comment]
@weiweishi I think the former is correct for Rails 5. Rails 4 uses rake

**Comment by botimer** [comment]
Something that will likely trip people trying to Dive In: the Sufia 7.2.0 dependencies do not use `flipflop` from the branch. So, when doing a fresh setup and the `sufia:install` generator is run, the config path setting explodes:

```
$ bundle exec rails g sufia:install -f
/bundle/gems/railties-4.2.7.1/lib/rails/railtie.rb:196:in `method_missing': undefined method `config_file=' for Flipflop::Engine:Class (NoMethodError)
	from /bundle/gems/sufia-7.2.0/lib/sufia/engine.rb:39:in `block in <class:Engine>'
```

Maybe a quick tag of 7.2.1?

**Comment by mjgiarlo** [comment]
@botimer What do you mean by:

> the Sufia 7.2.0 dependencies do not use `flipflop` from the branch

:question:

**Comment by botimer** [comment]
I mean that I added the 7.2.0 gem, ran bundle install, and then sufia:install, and the release version was in the bundle. This breaks the generator with the message above. I had to add flipflop according to the issue notes to my Gemfile to make things go.

**Comment by mjgiarlo** [comment]
@botimer Gotcha. I've updated the [7.2.0 release notes](https://github.com/projecthydra/sufia/releases/tag/v7.2.0) to try to make this clearer. Thanks for the note!
